eta/core/datasets.py

```diff
@@ -275,7 +277,7 @@ def sample_videos_to_images(
 
     _validate_stride_and_num_images(stride, num_images)
 
-    if num_images is not None:
+    if num_images is not None and stride is None:
         stride = _compute_stride(video_dataset, num_images, frame_filter)
         logger.info("Sampling video frames with stride %d", stride)
```

Comment:
Can the logger statement be changed? This is called when stride is None and there is no value for stride in `logger.info("Sampling video frames with stride %d", stride)`

Reply:
Not sure what you mean @rpinnaka . The stride is set in the line above this.

Reply:
@mattphotonman If the stride is originally set by the user, then the logger message will not be showing up. Can this be included for this case as well (which will be helpful for the user)?